src/test/java/org/opentripplanner/model/plan/PlanTestConstants.java

```diff
@@ -88,7 +90,7 @@ private static Place place(String name, double lat, double lon) {
             null
     );
     return Place.forStop(
-            stop, null, null
+            stop, null, null, new Locale("en", "US")
```

Comment:
Maybe extract `new Locale("en", "US")` as constant, and use the same constant for other test classes?

Reply:
Fixed to use constant here and other test files which used new Locale("en", "US").